lib/improver/spotdata/tests/test_neighbour_finding.py

```diff
@@ -0,0 +1,742 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm going to commit unit tests for the common functions with the extraction pull request (the functions are used in both routines).